WaterSkiBaan/Program.cs

```diff
@@ -148,6 +148,35 @@ static public void SportersGaanNaarInstructie(object sender, SportersEventArgs a
             }
         }
 
+        static public void ControleerPositie1(object sender, LijnEventArgs args)
+        {
+            LijnenInGebruik lijnenInGebruik = args.LijnenInGebruik;
+
+            if (lijnenInGebruik.Lijnen.Count > 0)
+            {
+
+                if (lijnenInGebruik.Lijnen.First.Value.Equals(null))
+                {
+                    lijnenInGebruik.NeemLijnInGebruik(lijnenInGebruik.LijnenVoorraad.Lijnen.Dequeue());
+                    lijnenInGebruik.Lijnen.First.Value.Sporter = _wachtrijStarten.Wachtrij.Dequeue();
+                }
```

Comment:
hey, my 2 cents.
Er kunnen max 5 lijnen rondkachelen. dus je hebt 5 posities. elke keer na het event LijnenVerplaatsen wordt de positie van elke in gebruik zijnde lijn geincrement met 1. jij checkt nu of er op de (edit: laatste moet dit zijn) 1e positie van de linked list dus [] [] [] [] -> [] <- (edit dit moet zijn ->[]<-[] [] [] []  geen object (lijn) aanwezig is. Lijkt mij dat je door de lijnen moet loopen en checken of 1tje zn attribuutvalue van positie op 1 heeft staan. een ijverige skier kan ook 28 rondjes varen en op een willekeurige plek zitten in de linkedlist met positie 1, dus je moet die hele lijst door, me dunkt. kijk er kan bijvoorbeeld ook 1 skieren rondkachelen en die kan na het event op positie 1 zitten het is immers een rondje die je maakt, op dat moment is het wel lullig als je dan net een nieuwe inhaakt (je linked list kan op first leeg zijn op dat moment).

Maar misschien zit ik ernaast.

edit: ik haalde de definitie laatste en eerste node door elkaar, maar niettemin op een andere node dan de eerste node in de linked list kan een skieer zitten die op dat moment positie 1 heeft, immers kunnen ze meerdere rondjes maken.

@arvenzis wat denk jij?

Reply:
Klinkt logisch. Volgens mij moet dit idd zo gemaakt worden. @saakee ben je het hiermee eens?